README.md

```diff
@@ -83,13 +80,56 @@ setting. In that case, try removing the `.next` cache folder.
 
 [feature-flags]: https://martinfowler.com/articles/feature-toggles.html
 
-## Style guide
+## Bootstrap + styled-components
+We use Bootstrap 4 for two purposes.
+1. Responsive grid system
+2. Style consistency across browsers ([reboot.css](https://getbootstrap.com/docs/4.0/content/reboot/))
 
-Project uses ESLint AirBnb style guide without semicolons. The `precommit` git hook will
-automatically run ESLint to check if the code complies with the rules. If it fails then it will not
-push to the repo.
+Our components are given custom styling using [styled-components](https://www.styled-components.com/) - a css-in-js library.
+
+## Fundraisingbox
+The service used to accept user donations is called [Fundraisingbox](https://www.fundraisingbox.com), which Masifunde has previous experience with.
+
+Masifunde's Fundraisingbox plan doesn't allow for using the API, but we get access to an iframe we embed on our site. We can embed 4 different iframe forms at a time.
+To configure the iframe, sign in to [Masifunde's Fundraisingbox admin interface](https://secure.fundraisingbox.com) and go to `Einstellungen > Spendenformular` and select the relevant Spendenformular in the dropdown.
+
+To fully customize the form, we hide most of the inputs in iframe, while rendering our own form fields outside of the iframe. We then [populate the iframe with these custom form field values](https://developer.fundraisingbox.com/v1.0/docs/form-prepopulation-api). This forces a reload and rerender of the iframe, which causes flickering. So we only do this when all custom form fields have values.
+
+### Einbettungsadresse (form embed url)
+The `Einbettungsadresse` should specify the full url of the page this form is used (e.g `https://www.masifunde.de/wie-sie-helfen/spenden`), and not just the host `https://www.masifunde.de`. 
```

Comment:
For terms like _Einbettungsadresse_ that are not code identifiers, it's better to use _emphasis_ or "quotes".